tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -295,9 +296,10 @@ Verify CPU And Memory Requests And Limits Are Defined For All Containers In All
     END
 
 Wait Until Operator Reverts "Grafana" To "Prometheus" In Rhods-Monitor-Federation
-    [Documentation]     wait and checks Operator have changed app grafana to prometheus
-    Sleep    10m    msg=wait to operator reverts the Changes
-    Wait Until Keyword Succeeds    5min    30s    Verify In Rhods-Monitor-Federation App Is    expected_app_name=prometheus
+    [Documentation]     Waits until rhods-operator reverts the configuration of rhods-monitor-federation,
+    ...    verifiying it has the default value ("prometheus")
+    Sleep    10m    msg=Waits until rhods-operator reverts the configuration of rhods-monitor-federation
```

Comment:
Maybe you could avoid using sleep in this keyword and just use "wait until keyword succeeds" for a longer period of time, so that you may save execution time.